pkgs/applications/misc/epdfview/default.nix

```diff
@@ -1,14 +1,16 @@
 { stdenv, fetchurl, fetchpatch, pkgconfig, gtk2, poppler }:
 
 stdenv.mkDerivation rec {
-  name = "epdfview-0.1.8";
+  name = "epdfview-${version}";
+  version = "0.1.8";
 
   src = fetchurl {
-    url = "http://trac.emma-soft.com/epdfview/chrome/site/releases/${name}.tar.bz2";
-    sha256 = "1w7qybh8ssl4dffi5qfajq8mndw7ipsd92vkim03nywxgjp4i1ll";
+    url = "mirror://debian/epdfview/epdfview_${version}.orig.tar.gz";
```

Comment:
I believe you need to include the `/pool/whatever` stuff in the url when using the debian mirrors.